data/Templates/Ccda/ValueSet/ValueSet.json

```diff
@@ -84,6 +90,9 @@
       "replaced": {
         "code": "amended"
       },
+      "ntered-in-error": {
```

Comment:
entered-in-error?

Reply:
Besides, are we making the same changes in ValueSets liquid templates?

Reply:
typo, I will fix it.

Reply:
> Besides, are we making the same changes in ValueSets liquid templates?

Yes, actually these changes are made to keep the same as ValueSets liquid templates.